shaku_axum/tests/module_interface.rs

```diff
@@ -0,0 +1,34 @@
+use shaku::{module, Component, HasComponent, HasProvider, Interface, Provider};
```

Comment:
This should copy over the comment from the similar test in shaku_actix (modified for axum as required).